content/docs/1.4.5/high-availability/recover-volume-when-read-only.md

```diff
@@ -0,0 +1,11 @@
+---
+  title: Recover Volume When It Becomes Read Only
+  weight: 1
```

Comment:
Remove needless indent for each variable definition.

Reply:
Fixed.